src/core/control/xojfile/LoadHandler.cpp

```diff
@@ -337,6 +337,24 @@ void LoadHandler::parseBgSolid() {
         bg.format = PageTypeHandler::getPageTypeFormatForString(style);
     }
 
+    if (bg.format == PageTypeFormat::Copy) {
+        /*
+         * PageTypeFormat::Copy is just a placeholder for the various background related menus, indicating that the
+         * background should be copied from another page.
+         * IT SHOULD NEVER APPEAR IN AN ACTUAL PAGE MODEL OR A FORTIORI SAVED FILE
+         *
+         * Before https://github.com/xournalpp/xournalpp/commit/257b935b3faa61f54a38f79dce734d666f7ab738
+         * a bug was creating files, the pages of which had background type PageTypeFormat::Copy.
+         * Those pages were displayed as PageTypeFormat::Plain.
```

Comment:
I don't think this description is entirely accurate, the bug is still present even after that commit.
```suggestion
         * Due to the bug https://github.com/xournalpp/xournalpp/issues/4127
         * it is possible for some older files to (incorrectly) contain pages with background type PageTypeFormat::Copy.
         * Such pages were displayed as PageTypeFormat::Plain.
```